browser-test/src/applicant_application_index.test.ts

```diff
@@ -141,9 +142,10 @@ describe('applicant program index page', () => {
     await applicantQuestions.clickNext()
     await applicantQuestions.submitFromReviewPage()
 
-    // Check that the first program now shows the previously answered text.
+    // Check that the other program now shows the previously answered text.
     await applicantQuestions.returnToProgramsFromSubmissionPage()
     await applicantQuestions.clickApplyProgramButton(otherProgramName)
-    await validateScreenshot(page, 'first-program-shows-previously-answered')
+    await applicantQuestions.validatePreviouslyAnsweredText()
```

Comment:
**suggestion (non-blocking):** It would be nice if we could verify that the correct question has the text instead of just any question has it. 

The `other` program only has 1 question, You could reverse how the programs are used so you're checking the program with 2 questions.

To verify you could follow this pattern.  Note that it uses a [ReferenceClass added to the dom element](https://sourcegraph.com/search?q=APPLICANT_NOT_ELIGIBLE_TEXT+repo%3Aciviform%2Fciviform%24&groupBy=repo) and [used in the .ts helper](https://sourcegraph.com/search?q=cf-applicant-not-eligible-text+repo%3Aciviform%2Fciviform%24&groupBy=repo)

https://sourcegraph.com/github.com/civiform/civiform@main/-/blob/browser-test/src/application_navigation.test.ts?L363

Reply:
Makes sense! Thanks for that example. I realized there is a bt-date attribute on the previously answered on element, so I used that and added those checks. Let me know what you think.

Reply:
Thank you!!

We should add a different class though instead of relying on that just happening to be there as it's effectively different from this need.

FWIW the .bt- classes are specifically added to indicate which items should be scrubbed to make screenshot validation stable, so dates, times, ids, etc.   bt stands for Browser Test